game/templates/game/landing.html

```
@@ -368,6 +369,71 @@ <h3 class="capability-title">REST API Architecture</h3>
       </div>
     </section>
 
+    <section id="faq" class="faq-section">
+      <div class="faq-container">
+        <h2 class="section-title">Frequently Asked Questions</h2>
+        <p class="faq-subtitle">Everything you need to know about the Checkora chess platform architecture and functionality.</p>
+        
+        <div class="faq-wrapper">
+          <div class="faq-item">
+            <button class="faq-question">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)